verilog/formatting/format_style.h

```diff
@@ -96,6 +96,9 @@ struct FormatStyle : public verible::BasicFormatStyle {
   // reduce the risk of harming already decent code.
   bool try_wrap_long_lines = true;
 
+  // Compact bit selection / indexing ranges
+  bool compact_non_declaration_ranges = true;
```

Comment:
Suggest naming this with what it includes, rather than what it excludes, something like: `compact_indexing_and_selections`, so we know what it is intended for (in case we make a mistake down the road and find that this implementation is too inclusive).  Also make it clear in the comment that this is concerning a subset of syntax inside []s.

Reply:
Done